File: test_api.js
fetch("https://scroll-and-sword-api.swordandscroll.workers.dev/", {
    method: "POST",
    headers: { "Content-Type": "application/json", "User-Agent": "Mozilla/5.0" },
    body: JSON.stringify({ theme: "medieval", hp: 10, act: 1, step: 1 })
})
    .then(async r => console.log(r.status, await r.text()))
    .catch(console.error);

File: test_api.py
import urllib.request, json
req = urllib.request.Request(
    'https://scroll-and-sword-api.swordandscroll.workers.dev/', 
    data=b'{"theme":"medieval","hp":10,"act":1,"step":1}', 
    headers={'Content-Type': 'application/json'}
)
try:
    res = urllib.request.urlopen(req)
    print(res.read().decode())
except Exception as e:
    print(e)
